### PR TITLE
FH-4049 Log Sync/Activity Warnings

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

File: library/src/main/java/com/feedhenry/sdk/sync/FHSyncConfig.java [hunk 1]
```diff
@@ -37,6 +37,7 @@
     private int mCrashCountWait = 10;
     private boolean mResendCrashedUpdates = true;
     private boolean mUseCustomSync = false;
+    private boolean supressActivityWarnings = false;
```
Comment (damienomurchu) [inline library/src/main/java/com/feedhenry/sdk/sync/FHSyncConfig.java]: typo - suppressActivityWarnings

File: build.gradle [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-        maven {
-          url 'https://maven.google.com'
-        } 
+        google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-beta2'
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
-
+        classpath 'com.android.tools.build:gradle:3.0.0-beta3'
```
Comment (damienomurchu) [inline build.gradle]: could do with a bump (3.0.0-beta7 is the current beta version)

File: library/build.gradle [hunk 3]
```diff
@@ -49,9 +49,9 @@ dependencies {
 
     testImplementation 'junit:junit:4.12'
 
-    androidTestImplementation 'com.android.support:support-annotations:25.3.0'
-    androidTestImplementation 'com.android.support.test:runner:0.5'
-    androidTestImplementation 'com.android.support.test:rules:0.5'
```
Comment (damienomurchu) [inline library/build.gradle]: probably do with a version bump, current versions:
- androidTestCompile 'com.android.support:support-annotations:26.1.0'
- androidTestCompile 'com.android.support.test:runner:1.0.1'
- androidTestCompile 'com.android.support.test:rules:1.0.1'

**Comment by danielpassos** [comment]
@damienomurchu #brotip since it's not going to master, you can merge it and send a PR for fix this (yourslef) or send a PR to Summers repo to make it fast ;)

**Comment by damienomurchu** [comment]
@danielpassos good point :)

**Comment by danielpassos** [comment]
@damienomurchu I'm going to fix all of this directly in `Android-O` branch

**Pull request merged**